src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -725,44 +855,190 @@ function wait_for_dynamo_frontend()
-  while [ ! -f "$DONE_MARKER" ]; do
-    exit_on_error
-    log "Waiting for frontend completion marker by polling $DONE_MARKER"
-    sleep 30
+  if [[ "${dynamo_args["connector"]}" != "lmcache" ]]; then
+    log "Connector is not lmcache. Skipping setup_lmcache"
+    return
+  fi
+
+  log "Setting up LMCache; installing LMCache using: uv pip install $lmcache_path"
+  local lmcache_path="${lmcache_config["repo"]}"
+  uv pip install -e $lmcache_path
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)